sdk/communication/azure-communication-chat/src/test/java/com/azure/android/communication/chat/chatClientTest.java

```diff
@@ -24,9 +25,17 @@
 
     private static final MockWebServer mockWebServer = new MockWebServer();
     private static final String BASE_URL = mockWebServer.url("/").toString();
-    private static AzureCommunicationChatServiceClient chatServiceClient =
-        new AzureCommunicationChatServiceClient.Builder()
+    private static UserAgentInterceptor userAgentInterceptor = new UserAgentInterceptor(
```

Comment:
Why do you need a userAgentInterceptor in this test? I can't see it being used

Reply:
It is just for testing the content of header if you break at some point of test, you can see the header from request.